app/models/issue_importer.rb

```diff
@@ -57,7 +57,18 @@ def initialize(github_client)
   #
   # Returns a Sawyer::Resource.
   def github_issue
-    @github_issue ||= github_client.issue "#{owner}/#{repository}", number
+    @github_issue ||= begin
+      path = "/repos/#{@owner}/#{@repository}/issues/#{@number}"
+      options = {}
+
+      if defined?(@updated_at)
+        options[:headers] = {
+          "If-Modified-Since" => @updated_at.httpdate
+        }
+      end
+
+      github_client.send(:request, :get, path, nil, options)
```

Comment:
Ohhhh, I misunderstood this. `""` will be `.blank?`, which isn't falsy, so the memoization will work right. Sorry for the noise. :grimacing: